File: packages/web-game-client/src/game/cards/gain_funds.ts
// packages/web-game-client/src/game/cards/gain_funds.ts

// packages/web-game-client/src/game/cards/gain_funds.ts

import { PlayerState } from '../../types';

/**
 * 「資金集め」カードの効果を適用します。
 * 資金が2増加します。
 * @param player 効果を適用するプレイヤーの状態
 */
export const applyGainFunds = (player: PlayerState): void => {
  player.funds += 2;
};

File: packages/api-server/app/game/cards/gain_funds.py
# packages/api-server/app/game/cards/gain_funds.py
from app.game.models import PlayerState

def apply_gain_funds(player: PlayerState) -> None:
    """
    「資金集め」カードの効果を適用します。
    資金が1増加します。
    """
    player.funds += 2
